<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStatusInfo</class>
 <widget class="QDialog" name="DialogStatusInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Status Info</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>411</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnUpdate">
      <property name="text">
       <string>VSTATUS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEsync">
      <property name="text">
       <string>ESYNC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt1Command"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
